blocks/autocompleters/README.md

```diff
@@ -0,0 +1,117 @@
+Autocompleters
```

Comment:
I have a feeling that this should be documented next to the original `Autocomplete` component. It is also breaking change as far as I understand. We should provide some notes how to update code when someone was using `wp.components.Autocomplete`.

Reply:
> We should provide some notes how to update code when someone was using wp.components.Autocomplete.

Where is a good place to provide such notes?

Reply:
> I plan to move the completer interface JSDoc into components/autocomplete/index.js. Sound reasonable?

Yes, this was my exactly my point. It's a general interface that should work with every custom `Autocomplete` component.